.github/workflows/docker-promote.yml

```diff
@@ -34,7 +34,7 @@ jobs:
         id: checkUserMember
         with:
           username: ${{ github.actor }}
-          team: 'approvers-ci'
+          team: 'approvers-infra'
```

Comment:
can we keep that under a CI owners permission level?

Reply:
Do you want to keep the `approvers-ci` groups, i.e., do not deprecate it? Should we then also use this group for certain files in `CODEOWNERS`?

Reply:
If we keep `approvers-ci`, what would be the structure?
* `approvers-ci` is a subset of `approvers-infra`, i.e., you can apply for `approvers-infra` only **or** you apply for `approvers-ci` and if accepted are approver for for `ci` and `infra`?